core/FrontController.php

```diff
@@ -332,10 +332,13 @@ public function init()
 
         $this->throwIfPiwikVersionIsOlderThanDBSchema();
 
-        if (empty($_GET['module'])
-            || empty($_GET['action'])
-            || $_GET['module'] !== 'Installation'
-            || !in_array($_GET['action'], array('getInstallationCss', 'getInstallationJs'))) {
+        $module = Common::getRequestVar('module', false, 'string');
```

Comment:
combination of default value  `false`  and  variable type  `string`  doesn't make much sense, as it will always result in an empty string as returned default value (as it will be casted to `string`)

Also wondering if we maybe should use `Piwik::getModule()` and `Piwik::getAction()` here

Reply:
> combination of default value false and variable type string doesn't make much sense, as it will always result in an empty string as returned default value (as it will be casted to string)

I know, it's still empty though and we use it in various places like this. 

Using Piwik:getModule etc makes sense 👍